tests/Auth0.AuthenticationApi.IntegrationTests/DatabaseConnectionTests.cs

```diff
@@ -63,17 +68,11 @@ public async Task Can_signup_user_and_change_password()
             signupUserResponse.Id.Should().NotBeNull();
             signupUserResponse.EmailVerified.Should().BeFalse();
             signupUserResponse.Email.Should().Be(signupUserRequest.Email);
-
-            // Change the user's Email address
-            var changePasswordRequest = new ChangePasswordRequest
```

Comment:
This test made no sense... password isn't even supposed to be on that request so now it's moved from that request object there's nothing to test.

Reply:
@damieng This was allowed previously. Probably now deprecated and allowed only with a legacy flag on the account. AFAIK today if you send the password, it will be ignored by the server but the request won't fail.

Reply:
Right, that's good context.  This test as it was was making the SDK more confusing to users in order to test that a server rejects things so I'm even more 👍 with this change.